apps/src/componentLibrary/common/contexts/DropdownContext.tsx

```diff
@@ -0,0 +1,27 @@
+import React, {createContext, useContext, useState, ReactNode} from 'react';
+
+const DropdownContext = createContext({
+  activeDropdownName: '',
+  /**
+   * This is a dummy function to satisfy the type checker
+   * */
+  setActiveDropdownName: (name: string) => {
+    return name as unknown as void;
+  },
+});
+
+export const useDropdownContext = () => useContext(DropdownContext);
+
+export const DropdownProvider: React.FC<{children: ReactNode}> = ({
```

Comment:
Not sure since all `DropdownProvider` do is adding `DropdownContext.Provider` around children components. `DropdownWrapper` might cause some confusions I guess since we don't have word Provider or Context in it.
Though, I think `DropdownProviderWrapper` or `DropdownContextWrapper` might be a good name giving the full context to the developer, leaning towards `DropdownProviderWrapper`. What do you think?

Reply:
Yeah, no strong opinion either way, especially since this should only be used internally by design system components anyway